tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -259,6 +259,16 @@ module.exports = {
 
       return this
     },
+    /**
+     *
+     * @param {string} fromName
+     */
+    downloadFile: function (fromName) {
+      return this.initAjaxCounters()
+        .waitForFileVisible(fromName)
+        .performFileAction(fromName, 'download')
```

Comment:
Should we put this into an `enum`-like? There's already 7 usages of this kind of strings.
```suggestion
        .performFileAction(fromName, FileAction.download)
```

Reply:
Moved to https://github.com/owncloud/phoenix/issues/2567